### PR TITLE
Login feature implemented finally(#139)

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Implements the login feature

#### Changes proposed in this pull request:

32441972-a852a1c4-c31f-11e7-9087-9a48d1b4c6aa

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,7 @@
   "version": "1.0.0",
   "description": "Chrome Extension of SUSI AI",
   "dependencies": {},
-  "devDependencies": {
```
Comment (ms10398) [inline package.json]: You deleted all dev dependencies which were needed @stealthanthrax you should at least run jshint tests once.

**Comment by stealthanthrax** [comment]
@ms10398 ,for some reason the files were not added in the staging area,i have fixed them ,please have a look

File: src/options.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+});
+logoutButton.addEventListener("click", function logout(e){
+    e.preventDefault();
+	chrome.storage.sync.remove("messagesHistory");
+	chrome.storage.sync.remove("loggedUser");
+});
+
+document.addEventListener("DOMContentLoaded");
+
```
Comment (ms10398) [inline src/options.js]: Please remove the blank lines @stealthanthrax

File: src/settings.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		<form id="login" class="col-sm-8" >
+			<input type="email" id="username" placeholder="Email">
+			<br>
+			<input type="password" id="password" placeholder="Password">
+			<br>
+			<br>
+			<button type="submit" name="Login" id="loginbutton" value="Login">Login</button>
+			<br>
+			<p class="col-12" style="margin-left: 100px;">
+               <a href="https://chat.susi.ai/" style="color: #fff; ">New User? Visit our website to sign up for free.</a>
```
Comment (ms10398) [inline src/settings.html]: Can you remove `for free` as everything here is Open Source.

File: src/options.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+var logoutButton = document.getElementById("logout");
+var loginButton = document.getElementById("loginbutton");
+var noLoggedInBlock = document.getElementById("nologgedin");
+var loggedInBlock = document.getElementById("loggedin");
+var accessToken = "";
+var userEmail = "";
+var time = "";
+var BASE_URL = "https://api.susi.ai";
+//var messagesHistory=[];
+function showLoggedInBlock(show){
```
Comment (ms10398) [inline src/options.js]: Is there any need of this block @stealthanthrax

File: src/options.js [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+			}
+			if (status === "timeout") {
+				msg = "Please check your internet connection";
+			}
+			alert(msg);
+		}
+	});
+
+});
+logoutButton.addEventListener("click", function logout(e){
```
Comment (ms10398) [inline src/options.js]: Logout button is not working! But I will fix all these issues in a PR.

File: src/settings.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+            </p>
+		</form>
+	</div>
+	<div class="loggedin" id="loggedin">
```
Comment (ms10398) [inline src/settings.html]: The block of code down from here is unnecessary as it doesnt provide any functionality! 
Please remove it!

File: src/options.js (hunk 5 above)
Comment (stealthanthrax) [reply src/options.js]: I have fixed it @ms10398

File: src/options.js (hunk 4 above)
Comment (stealthanthrax) [reply src/options.js]: @ms10398 , I can't understand which block you are referring to, if you are referring to the comment on line 13 , it was a mistake and ill remove it

File: src/settings.html (hunk 6 above)
Comment (stealthanthrax) [reply src/settings.html]: @ms10398 the code below line 31 provides the logout button and links the js files.So what should i remove?

File: src/options.js (hunk 4 above)
Comment (ms10398) [reply src/options.js]: showLoggedin and password change as this functions are not there so please remove it as it has no use for the login feature!!

File: src/settings.html (hunk 6 above)
Comment (ms10398) [reply src/settings.html]: Yeah but as that button doesnt provides any functionality thats why i said!

**Pull request merged**